src/clj/witan/send/model/output.clj

```diff
@@ -258,7 +272,19 @@
            simulations simulated-transitions]}
    {:keys [run-outputs run-charts project-dir output-dir settings-to-exclude-in-charts
            keep-temp-files? use-confidence-bound-or-interval population-file]}]
-  (let [transform-transitions (->> transitions
+  (println "Outputting Results")
+
+  ;; Output the rest of the results
+  (let [dir (str project-dir "/" output-dir)
+        _ (when (not (.isDirectory (io/file dir)))
+            (.mkdir (io/file dir)))
+        ;; Output the big simulation results
+        _ (a/thread (output-simulated-transitions (str project-dir "/" output-dir) "simulated-transitions" @simulated-transitions))
```

Comment:
I'm not sure where there is a `thread` here for the whole process? I think I understand the `thread` within `output-simulated-transitions`

Reply:
Oh I think it's because there are more threads downstream